lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -467,4 +467,19 @@ protected function transformUser($data) {
     ];
   }
 
+  /**
+   * Set the global user to a different account based on Northstar id.
+   *
+   * @param  String $northstarUserId
+   */
+  protected function asNorthstarUser($northstarUserId) {
+    if (!user_access('view any reportback')) return;
+
+    $userOverride = dosomething_user_get_user_by_northstar_id($northstarUserId);
+    if ($userOverride) {
+      global $user;
+      $user = $userOverride;
```

Comment:
Does this actually override the `global` user? Or do you need to `return` this user back as the thing to actually use moving forward?